docs/develop/python/message-passing.mdx

```diff
@@ -362,54 +501,24 @@ Use static definitions as the primary way of structuring your Workflows.
 Reserve Dynamic Handlers for cases where the handler names are not known at compile time and need to be looked up dynamically at runtime.
 They are meant to handle edge cases and act as a catch-all, not as the main way of invoking logic.
 
-:::
-
-### Set a Dynamic Signal {#set-a-dynamic-signal}
-
-**How to set a Dynamic Signal**
+### Set a Dynamic Signal, Query, or Update handler {#set-a-dynamic-signal}
 
 A Dynamic Signal in Temporal is a Signal that is invoked dynamically at runtime if no other Signal with the same input is registered.
 A Signal can be made dynamic by adding `dynamic=True` to the `@signal.defn` decorator.
```

Comment:
```suggestion
A Signal can be made dynamic by adding `dynamic=True` to the `@workflow.signal` decorator.
```

(just saw when translating these docs)

Reply:
Good catch! (Pre-existing error in docs. d48ef208d780fa163eec6f022b03c13b2df11ae8)